packages/DataTable/src/components/InfiniteLoader/InfiniteLoader.tsx

```diff
@@ -3,8 +3,11 @@
 import React from "react";
 import { ListChildComponentProps, VariableSizeList } from "react-window";
 import ReactInfiniteLoader from "react-window-infinite-loader";
-import calculateRowHeight from "../../helpers/calculateRowHeight";
-import { TableDataItemType } from "../../types";
+import RowHeightHelper from "../../helpers/rowHeightHelper";
+import { useReactTableContext } from "../ReactTableContext";
+import { TableColumnsWidth, TableDataItemType } from "../../types";
+
+const calculateRowHeight = new RowHeightHelper().calculate;
```

Comment:
You shouldn't use class methods as standalone callbacks. 
It may work in this case, but it is will fails if you use `this.` and generally is discouraged.

```js
class RowHeightHelper {
  constructor() {
    this.myVar = 42;
  }

  getMyVar() {
    return this.myVar;
  }
}

const helper = new RowHeightHelper();
const getMyVar = helper.getMyVar;

console.log(helper.getMyVar()); // 42
console.log(getMyVar()); // Cannot read properties of undefined (reading 'myVar')
``` 

So, 

```js
cons  rowHeightHelper = new RowHeightHelper();


rowHeightHelper.calculateHowHeight()
```